lib/ab/configuration.rb

```diff
@@ -0,0 +1,22 @@
+module Ab
+  class << self
+    attr_writer :config
+
+    def config
+      @config ||= Config.new
+    end
+
+    def configure
+      @config ||= Config.new
+      yield(@config)
+    end
+  end
+
+  class Config
+    attr_accessor :logger
+
+    def initialize
+      @logger = Logger.new(nil)
```

Comment:
To be honest when I was writing it, I just looked at what Avdi did here https://github.com/avdi/naught#whats-that-supposed-to-mean :)